MobileBuy/buy/src/androidTest/java/com/shopify/buy/service/BuyTest.java

```diff
@@ -535,6 +549,21 @@ private Address getShippingAddress() {
         return shippingAddress;
     }
 
+    private Address getShippingAddressTwo() {
+        Address shippingAddress = new Address();
```

Comment:
Could you create the second one with `getShippingAddress()` and then just change the city, since that's the only thing changing?